src/Apcu/bootstrap.php

```diff
@@ -11,10 +11,14 @@
 
 if (!extension_loaded('apc') && !extension_loaded('apcu')) {
     return;
-}
-
-if (!function_exists('apcu_add')) {
+} else if (extension_loaded ('Zend Data Cache')) {
+    // Zend Data Cache refuses to load in the CLI, regardless of the configuration
+    if ('cli' != PHP_SAPI && false !== ini_get('zend_datacache.apc_compatibility')) {
+        require_once __DIR__ . DIRECTORY_SEPARATOR . 'bootstrap_zend.php';
```

Comment:
I'm open to suggestions.  I was trying to keep the APCUIterator, as well as trying to avoid the overhead associated with the Zend path on non-zend servers.  I could have moved it to composer.json and loaded both files that way, or I could have combined the functionality into a single file, with different function definitions depending on which path was taken.

I also didn't want to force APC to have to run the compatibility code, either.